web/src/beta/components/fields/ToggleField/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import Property from "@reearth/beta/components/fields";
+import Toggle from "@reearth/beta/components/Toggle";
+
+type ToggleProps = React.ComponentProps<typeof Toggle>;
```

Comment:
Instead of this, can you just export Props from components/Toggle and import here from that component? 🙏🏻

Reply:
That was what I initially did but then I thought rather than having multiple exports, this made more sense but I understand that that's more `simpler` and easy to understand.